mcts/src/main/resources/MctsAuditModification.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE motech_data_services.nms_mcts_audit CHANGE importDate endImportDate DATE NULL;
```

Comment:
This really isn't safe. What is the reason to change columns names? You will impact all reports, potentially ETL (though we don't have any) and this will force to you take downtime across all three instances.

Also, this break back compat with previous data models and is not good service design.